### PR TITLE
Remove old manage poll controller + Various bug fixes. 

### PR DESCRIPTION
**Pull request by stevenhillcox**

### CODE REVIEW

File: VotingApplication/VotingApplication.Web/Api/Controllers/ManageVoterController.cs [hunk 1]
```diff
@@ -59,10 +59,14 @@ private static ManageVoteResponseModel CreateManageVoteResponse(Ballot ballot)
             var model = new ManageVoteResponseModel
             {
                 BallotManageGuid = ballot.ManageGuid,
-                VoterName = ballot.VoterName,
                 Votes = new List<VoteResponse>()
             };
 
+            if (ballot.VoterName != null)
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web/Api/Controllers/ManageVoterController.cs]: Can we use the String.IsNullOrWhitespace for this?

File: VotingApplication/VotingApplication.Web/Api/Controllers/PollResultsController.cs [hunk 2]
```diff
@@ -87,7 +87,11 @@ private VoteRequestResponseModel VoteToModel(Vote vote, Poll poll)
                 model.OptionName = vote.Option.Name;
             }
 
-            model.VoterName = poll.NamedVoting ? vote.Ballot.VoterName : "Anonymous User";
+            if (vote.Ballot.VoterName != null)
+            {
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web/Api/Controllers/PollResultsController.cs]: ditto

File: VotingApplication/VotingApplication.Web/Api/Controllers/PollVoteController.cs [hunk 3]
```diff
@@ -59,14 +59,9 @@ private static VoteRequestResponseModel VoteToModel(Vote vote, Poll poll)
...
                 model.OptionName = vote.Option.Name;
-                model.VoterName = vote.Ballot.VoterName;
             }
 
-            if (!poll.NamedVoting || vote.Ballot.VoterName == null)
-            {
-                model.VoterName = "Anonymous User";
-            }
-            else
+            if (poll.NamedVoting && vote.Ballot.VoterName != null)
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web/Api/Controllers/PollVoteController.cs]: ditto

**Comment by tpkelly** [comment]
:+1:

**Comment by tpkelly** [comment]
:+1:

**Comment by JDawes-ScottLogic** [comment]
:+1: :shipit:

**Pull request merged**